packages/jest-cli/src/runTest.js

```diff
@@ -13,74 +13,91 @@ import type {Path, Config} from 'types/Config';
 import type {TestResult} from 'types/TestResult';
 import type {Resolver} from 'types/Resolve';
 
+const fs = require('fs');
+const {getTestEnvironment} = require('jest-config');
 const BufferedConsole = require('./lib/BufferedConsole');
+const promisify = require('./lib/promisify');
 const {
   Console,
   NullConsole,
   setGlobal,
 } = require('jest-util');
 const getConsoleOutput = require('./reporters/getConsoleOutput');
```

Comment:
Please keep the requires consistent. Here is how it should work with Jest:

```
const Capitalized = require('MyClass');

const {getTestEnvironment} = require('jest-config');
const lowercase = require('myfunction');
```

We have this weird spacing rule at FB, let's keep that for now :)